en/hosting_guide/migration.rst

```diff
@@ -0,0 +1,67 @@
+Migration
+=========
+
+To know more about migration see `mongodb migration`_.
+
+MongoDB Migration
+-----------------
+
+After having install MongoDBMigrationsBundle, it is possible to list the different allowed commands with :
```

Comment:
You should rephrase it with : the MongoDBMigrationsBundle come with a set of command available under the name space `mongodb:migrations`